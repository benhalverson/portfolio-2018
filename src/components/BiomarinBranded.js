import React from 'react';
import BiomarinCaseStudies from '../images/BiomarinCaseStudies.png';
import BiomarinCaseStudy from '../images/BiomarinCaseStudy.png';
import BiomarinCaseStudyWithAnswer from '../images/BiomarinCaseStudyWithAnswer.png';
import BiomarinChart from '../images/BiomarinChart.png';
import BiomarinCaseStudyWithWrongAnswer from '../images/BiomarineCaseStudyWithWrongAnswer.png';
import BiomarinHomePage from '../images/BiomarinHomePage.png';
import BiomarinIntroduction from '../images/BiomarinIntroduction.png';
import BiomarinRatingScale from '../images/BiomarinRatingScale.png';
import BiomarinSummary from '../images/BiomarinSummary.png';
import BiomarinVideoModal from '../images/BiomarinVideoModal.png';
import BiomarinVideoModal2 from '../images/BiomarinVideoModal2.png';
import BiomarinVideoPlayer from '../images/BiomarinVideoPlayer.png';

const BiomarinBranded = () => (
  <div>
    <div className="title">BioMarin CLN2 iPad App</div>
        <div className="body">
          <p className="description">
            I was the lead on this project when I was still realtively new to
            Viscira. The requirements from the client was unique. They had their
            own enterprsie iPad container that could accept webpages. I
            recommended that we use Angular 2 with the CLI to quickly build out
            the pages and router needed to move between the nav and subnavpages.
          </p>
          <div className="screenshots">
            <img className="col" src={BiomarinHomePage} alt="" />
            <img className="col" src={BiomarinIntroduction} alt="" />
            <img className="col" src={BiomarinCaseStudy} alt="" />
            <img className="col" src={BiomarinChart} alt="" />
            <img className="col" src={BiomarinRatingScale} alt="" />
            <img className="col" src={BiomarinCaseStudies} alt="" />
            <img className="col" src={BiomarinCaseStudyWithAnswer} alt="" />
            <img
              className="col"
              src={BiomarinCaseStudyWithWrongAnswer}
              alt=""
            />
            <img className="col" src={BiomarinVideoModal} alt="" />
            <img className="col" src={BiomarinVideoModal2} alt="" />
            <img className="col" src={BiomarinVideoPlayer} alt="" />
            <img className="col" src={BiomarinSummary} alt="" />
          </div>
        </div>
  </div>
);

export default BiomarinBranded;