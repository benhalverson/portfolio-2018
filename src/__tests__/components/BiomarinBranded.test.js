import React from 'react';
import { shallow } from 'enzyme';
import BiomarinBranded from '../../components/BiomarinBranded';
test('it should render the BiomarinBranded component', () => {
  const wrapper = shallow(<BiomarinBranded/>);
  expect(wrapper).toMatchSnapshot();
});