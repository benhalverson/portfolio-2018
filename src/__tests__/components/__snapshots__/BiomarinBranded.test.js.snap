// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render the BiomarinBranded component 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <BiomarinBranded />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="title"
>
          BioMarin CLN2 iPad App
</div>,
        <div
          className="body"
>
          <p
                    className="description"
          >
                    I was the lead on this project when I was still realtively new to Viscira. The requirements from the client was unique. They had their own enterprsie iPad container that could accept webpages. I recommended that we use Angular 2 with the CLI to quickly build out the pages and router needed to move between the nav and subnavpages.
          </p>
          <div
                    className="screenshots"
          >
                    <img
                              alt=""
                              className="col"
                              src="BiomarinHomePage.png"
                    />
                    <img
                              alt=""
                              className="col"
                              src="BiomarinIntroduction.png"
                    />
                    <img
                              alt=""
                              className="col"
                              src="BiomarinCaseStudy.png"
                    />
                    <img
                              alt=""
                              className="col"
                              src="BiomarinChart.png"
                    />
                    <img
                              alt=""
                              className="col"
                              src="BiomarinRatingScale.png"
                    />
                    <img
                              alt=""
                              className="col"
                              src="BiomarinCaseStudies.png"
                    />
                    <img
                              alt=""
                              className="col"
                              src="BiomarinCaseStudyWithAnswer.png"
                    />
                    <img
                              alt=""
                              className="col"
                              src="BiomarineCaseStudyWithWrongAnswer.png"
                    />
                    <img
                              alt=""
                              className="col"
                              src="BiomarinVideoModal.png"
                    />
                    <img
                              alt=""
                              className="col"
                              src="BiomarinVideoModal2.png"
                    />
                    <img
                              alt=""
                              className="col"
                              src="BiomarinVideoPlayer.png"
                    />
                    <img
                              alt=""
                              className="col"
                              src="BiomarinSummary.png"
                    />
          </div>
</div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "BioMarin CLN2 iPad App",
          "className": "title",
        },
        "ref": null,
        "rendered": "BioMarin CLN2 iPad App",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p
              className="description"
>
              I was the lead on this project when I was still realtively new to Viscira. The requirements from the client was unique. They had their own enterprsie iPad container that could accept webpages. I recommended that we use Angular 2 with the CLI to quickly build out the pages and router needed to move between the nav and subnavpages.
</p>,
            <div
              className="screenshots"
>
              <img
                            alt=""
                            className="col"
                            src="BiomarinHomePage.png"
              />
              <img
                            alt=""
                            className="col"
                            src="BiomarinIntroduction.png"
              />
              <img
                            alt=""
                            className="col"
                            src="BiomarinCaseStudy.png"
              />
              <img
                            alt=""
                            className="col"
                            src="BiomarinChart.png"
              />
              <img
                            alt=""
                            className="col"
                            src="BiomarinRatingScale.png"
              />
              <img
                            alt=""
                            className="col"
                            src="BiomarinCaseStudies.png"
              />
              <img
                            alt=""
                            className="col"
                            src="BiomarinCaseStudyWithAnswer.png"
              />
              <img
                            alt=""
                            className="col"
                            src="BiomarineCaseStudyWithWrongAnswer.png"
              />
              <img
                            alt=""
                            className="col"
                            src="BiomarinVideoModal.png"
              />
              <img
                            alt=""
                            className="col"
                            src="BiomarinVideoModal2.png"
              />
              <img
                            alt=""
                            className="col"
                            src="BiomarinVideoPlayer.png"
              />
              <img
                            alt=""
                            className="col"
                            src="BiomarinSummary.png"
              />
</div>,
          ],
          "className": "body",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "I was the lead on this project when I was still realtively new to Viscira. The requirements from the client was unique. They had their own enterprsie iPad container that could accept webpages. I recommended that we use Angular 2 with the CLI to quickly build out the pages and router needed to move between the nav and subnavpages.",
              "className": "description",
            },
            "ref": null,
            "rendered": "I was the lead on this project when I was still realtively new to Viscira. The requirements from the client was unique. They had their own enterprsie iPad container that could accept webpages. I recommended that we use Angular 2 with the CLI to quickly build out the pages and router needed to move between the nav and subnavpages.",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <img
                  alt=""
                  className="col"
                  src="BiomarinHomePage.png"
/>,
                <img
                  alt=""
                  className="col"
                  src="BiomarinIntroduction.png"
/>,
                <img
                  alt=""
                  className="col"
                  src="BiomarinCaseStudy.png"
/>,
                <img
                  alt=""
                  className="col"
                  src="BiomarinChart.png"
/>,
                <img
                  alt=""
                  className="col"
                  src="BiomarinRatingScale.png"
/>,
                <img
                  alt=""
                  className="col"
                  src="BiomarinCaseStudies.png"
/>,
                <img
                  alt=""
                  className="col"
                  src="BiomarinCaseStudyWithAnswer.png"
/>,
                <img
                  alt=""
                  className="col"
                  src="BiomarineCaseStudyWithWrongAnswer.png"
/>,
                <img
                  alt=""
                  className="col"
                  src="BiomarinVideoModal.png"
/>,
                <img
                  alt=""
                  className="col"
                  src="BiomarinVideoModal2.png"
/>,
                <img
                  alt=""
                  className="col"
                  src="BiomarinVideoPlayer.png"
/>,
                <img
                  alt=""
                  className="col"
                  src="BiomarinSummary.png"
/>,
              ],
              "className": "screenshots",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "className": "col",
                  "src": "BiomarinHomePage.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "className": "col",
                  "src": "BiomarinIntroduction.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "className": "col",
                  "src": "BiomarinCaseStudy.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "className": "col",
                  "src": "BiomarinChart.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "className": "col",
                  "src": "BiomarinRatingScale.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "className": "col",
                  "src": "BiomarinCaseStudies.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "className": "col",
                  "src": "BiomarinCaseStudyWithAnswer.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "className": "col",
                  "src": "BiomarineCaseStudyWithWrongAnswer.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "className": "col",
                  "src": "BiomarinVideoModal.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "className": "col",
                  "src": "BiomarinVideoModal2.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "className": "col",
                  "src": "BiomarinVideoPlayer.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "className": "col",
                  "src": "BiomarinSummary.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="title"
>
            BioMarin CLN2 iPad App
</div>,
          <div
            className="body"
>
            <p
                        className="description"
            >
                        I was the lead on this project when I was still realtively new to Viscira. The requirements from the client was unique. They had their own enterprsie iPad container that could accept webpages. I recommended that we use Angular 2 with the CLI to quickly build out the pages and router needed to move between the nav and subnavpages.
            </p>
            <div
                        className="screenshots"
            >
                        <img
                                    alt=""
                                    className="col"
                                    src="BiomarinHomePage.png"
                        />
                        <img
                                    alt=""
                                    className="col"
                                    src="BiomarinIntroduction.png"
                        />
                        <img
                                    alt=""
                                    className="col"
                                    src="BiomarinCaseStudy.png"
                        />
                        <img
                                    alt=""
                                    className="col"
                                    src="BiomarinChart.png"
                        />
                        <img
                                    alt=""
                                    className="col"
                                    src="BiomarinRatingScale.png"
                        />
                        <img
                                    alt=""
                                    className="col"
                                    src="BiomarinCaseStudies.png"
                        />
                        <img
                                    alt=""
                                    className="col"
                                    src="BiomarinCaseStudyWithAnswer.png"
                        />
                        <img
                                    alt=""
                                    className="col"
                                    src="BiomarineCaseStudyWithWrongAnswer.png"
                        />
                        <img
                                    alt=""
                                    className="col"
                                    src="BiomarinVideoModal.png"
                        />
                        <img
                                    alt=""
                                    className="col"
                                    src="BiomarinVideoModal2.png"
                        />
                        <img
                                    alt=""
                                    className="col"
                                    src="BiomarinVideoPlayer.png"
                        />
                        <img
                                    alt=""
                                    className="col"
                                    src="BiomarinSummary.png"
                        />
            </div>
</div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "BioMarin CLN2 iPad App",
            "className": "title",
          },
          "ref": null,
          "rendered": "BioMarin CLN2 iPad App",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p
                className="description"
>
                I was the lead on this project when I was still realtively new to Viscira. The requirements from the client was unique. They had their own enterprsie iPad container that could accept webpages. I recommended that we use Angular 2 with the CLI to quickly build out the pages and router needed to move between the nav and subnavpages.
</p>,
              <div
                className="screenshots"
>
                <img
                                alt=""
                                className="col"
                                src="BiomarinHomePage.png"
                />
                <img
                                alt=""
                                className="col"
                                src="BiomarinIntroduction.png"
                />
                <img
                                alt=""
                                className="col"
                                src="BiomarinCaseStudy.png"
                />
                <img
                                alt=""
                                className="col"
                                src="BiomarinChart.png"
                />
                <img
                                alt=""
                                className="col"
                                src="BiomarinRatingScale.png"
                />
                <img
                                alt=""
                                className="col"
                                src="BiomarinCaseStudies.png"
                />
                <img
                                alt=""
                                className="col"
                                src="BiomarinCaseStudyWithAnswer.png"
                />
                <img
                                alt=""
                                className="col"
                                src="BiomarineCaseStudyWithWrongAnswer.png"
                />
                <img
                                alt=""
                                className="col"
                                src="BiomarinVideoModal.png"
                />
                <img
                                alt=""
                                className="col"
                                src="BiomarinVideoModal2.png"
                />
                <img
                                alt=""
                                className="col"
                                src="BiomarinVideoPlayer.png"
                />
                <img
                                alt=""
                                className="col"
                                src="BiomarinSummary.png"
                />
</div>,
            ],
            "className": "body",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "I was the lead on this project when I was still realtively new to Viscira. The requirements from the client was unique. They had their own enterprsie iPad container that could accept webpages. I recommended that we use Angular 2 with the CLI to quickly build out the pages and router needed to move between the nav and subnavpages.",
                "className": "description",
              },
              "ref": null,
              "rendered": "I was the lead on this project when I was still realtively new to Viscira. The requirements from the client was unique. They had their own enterprsie iPad container that could accept webpages. I recommended that we use Angular 2 with the CLI to quickly build out the pages and router needed to move between the nav and subnavpages.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <img
                    alt=""
                    className="col"
                    src="BiomarinHomePage.png"
/>,
                  <img
                    alt=""
                    className="col"
                    src="BiomarinIntroduction.png"
/>,
                  <img
                    alt=""
                    className="col"
                    src="BiomarinCaseStudy.png"
/>,
                  <img
                    alt=""
                    className="col"
                    src="BiomarinChart.png"
/>,
                  <img
                    alt=""
                    className="col"
                    src="BiomarinRatingScale.png"
/>,
                  <img
                    alt=""
                    className="col"
                    src="BiomarinCaseStudies.png"
/>,
                  <img
                    alt=""
                    className="col"
                    src="BiomarinCaseStudyWithAnswer.png"
/>,
                  <img
                    alt=""
                    className="col"
                    src="BiomarineCaseStudyWithWrongAnswer.png"
/>,
                  <img
                    alt=""
                    className="col"
                    src="BiomarinVideoModal.png"
/>,
                  <img
                    alt=""
                    className="col"
                    src="BiomarinVideoModal2.png"
/>,
                  <img
                    alt=""
                    className="col"
                    src="BiomarinVideoPlayer.png"
/>,
                  <img
                    alt=""
                    className="col"
                    src="BiomarinSummary.png"
/>,
                ],
                "className": "screenshots",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "className": "col",
                    "src": "BiomarinHomePage.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "className": "col",
                    "src": "BiomarinIntroduction.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "className": "col",
                    "src": "BiomarinCaseStudy.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "className": "col",
                    "src": "BiomarinChart.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "className": "col",
                    "src": "BiomarinRatingScale.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "className": "col",
                    "src": "BiomarinCaseStudies.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "className": "col",
                    "src": "BiomarinCaseStudyWithAnswer.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "className": "col",
                    "src": "BiomarineCaseStudyWithWrongAnswer.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "className": "col",
                    "src": "BiomarinVideoModal.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "className": "col",
                    "src": "BiomarinVideoModal2.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "className": "col",
                    "src": "BiomarinVideoPlayer.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "className": "col",
                    "src": "BiomarinSummary.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
